pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/PmdRuleCataloger.java

```diff
@@ -65,10 +65,10 @@ class PmdRuleCataloger {
 	void catalogRules() {
 		//Check for custom rules and process them
 
-		// STEP 1: Identify all of the ruleset and category files for each language we're looking at.
+		// Identify all the ruleset and category files for each language we're looking at.
 		extractRules();
 
-		// STEP 2: Process the category files to derive category and rule representations.
+		// STEP: Process the category files to derive category and rule representations.
```

Comment:
Nit: Remove "STEP:" so the comments are consistent.

Reply:
ok - good catch.